variables.tf

```diff
@@ -154,3 +154,19 @@ variable "grafana_service_accounts" {
     "monitoring:monitoring-monitoring-prometheus-node-exporter"
   ]
 }
+
+variable "create_ub_cosign_iam_role" {
+  description = "Flag to create Cosign IAM role"
+  default     = false
+  type        = bool
+}
+
+variable "iam_role_path" {
+  default = "/delegatedadmin/developer/"
+  type    = string
+}
+
+variable "iam_role_permissions_boundary" {
+  default = "arn:aws:iam::373346310182:policy/cms-cloud-admin/developer-boundary-policy"
```

Comment:
```suggestion
```

Reply:
I know this is set as default elsewhere, but it probably shouldn't be.  We won't be able to create roles unless this is set.  If we decide to keep a default, we should set it as empty string.

Reply:
Updated as suggested.